dev/files/authn-jenkins/policy.yml

```diff
@@ -0,0 +1,15 @@
+- !policy
+  id: conjur/authn-jenkins/prod
+  body:
+  - !webservice
+  - !group clients
+  # these will be used to connect to the jenkins server and see if a specific job is running
+  - !variable jenkinsURL
```

Comment:
I like this idea, I will try to implement this. In this example is 'ci/jenkins/url' an absolute path or a relative path to the webservice? It would be nice to have it absolute so we could onboard the jenkins account into cyberark and have cyberark automatically rotate the credential and then update via the synchronizer. Also do permissions need to be granted to the webservice to fetch specific secrets?

Reply:
An absolute path feels like the best option. We could always force it with a leading `/`.  A webservice should be entitled as part of the policy, otherwise resources can gain access to credentials without explicitly being granted access.

Reply:
So this behavior is possible but it requires jumping through some hoops.
```ruby
url_id = variables.annotation("url") 
username_id = variables.annotation("username")
password_id = variables.annotation("password")

url = variables.resource_class["#{input.account}:variable:#{url_id}"].secret.value
username = variables.resource_class["#{input.account}:variable:#{username_id}"].secret.value
password = variables.resource_class["#{input.account}:variable:#{password_id}"].secret.value
```

Lets discuss this further